gravatar/src/main/java/com/gravatar/restapi/models/Error.kt

```diff
@@ -0,0 +1,56 @@
+/**
+ *
+ * Please note:
+ * This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * Do not edit this file manually.
+ *
+ */
+package com.gravatar.restapi.models
+
+import com.google.gson.annotations.SerializedName
+import java.util.Objects
+
+/**
+ *
+ *
```

Comment:
⛏ Not a direct issue of the PR, but we can add the class description in the specification file.

Reply:
I've asked Diego about this.